library/src/main/kotlin/com/connectrpc/protocols/GRPCWebInterceptor.kt

```diff
@@ -259,7 +230,6 @@ internal class GRPCWebInterceptor(
         if (headers.keys.none { it.equals(GRPC_WEB_USER_AGENT, ignoreCase = true) }) {
             headers[GRPC_WEB_USER_AGENT] = listOf("grpc-kotlin-connect/${ConnectConstants.VERSION}")
         }
-        headers[GRPC_TE_HEADER] = listOf("trailers")
```

Comment:
gRPC-web specifically doesn't supported trailers - we shouldn't send this header.